src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as fhirtypes from './fhirtypes';
+export * as fshtypes from './fshtypes';
```

Comment:
I expect we'll need more than this for the FSH Playground.  Can you talk to @SamCitron and @jafeltra to determine what they need exported to support the FSH Playground?

Reply:
I thought you used `Processing`, no?

Reply:
The `playgroundApp` function uses functions from `Processing`, but I had thought these exports were for things that are being directly imported in the separate project. If that's not the case, then I can get a more extensive list of all the functions that `playgroundApp` is using.

Reply:
How does the playground use functions from `Processing` without importing them?

Reply:
Also, we wouldn't need to export each function.  We'd probably just export the whole `utils` module.

Reply:
Ah okay I see where the disconnect is here. Right now, the changes we made for the playground all live in functions that are in SUSHI. So the playground only imports that one function, `playgroundApp`. @cmoesel and I discussed this separately and maybe this is something we'd want to reconsider. If we were to export all the functions that the `playgroundApp` function is currently using if it were to be moved into the Playground repo we'd need the following modules also exported:

- `utils` (for various Processing functions, the logger, and the Type enum)
- `export` (for the `exportFHIR` function)
- `import` (for the `RawFsh` class)
- `fhirDefs` (for the `FHIRDefinitions` class)

This shift might be something larger we want to discuss, so I'm not sure if it's easier to hold off on exporting these or if it's easier to just export them all now even if they don't get used.